### PR TITLE
feat:implemented multiplication func

### PR DESCRIPTION
**Pull request by manvithalva12**
multiplication function

### CODE REVIEW

File: practice.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+function multiply(a,b){
```
Comment (albin1616) [inline practice.js:1]: add js doc

Comment (Rajharsh2) [inline practice.js:1]: put some comments for better explaination

**Review by Rajharsh2 — CHANGES_REQUESTED** [review]

**Review by albin1616 — CHANGES_REQUESTED** [review]

**Review by Rajharsh2 — APPROVED** [review]
lgtm

**Review by albin1616 — APPROVED** [review]

File: practice.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//this function multiplies 2 integers
+//**
+ * 
+ * @param {Number} a must be a number
+ * @param {Number} b must be a number
+ * @returns the results 
+ */
```
Comment (Rajharsh2) [inline practice.js:7]: add cmment here to explain function

**Review by Rajharsh2 — CHANGES_REQUESTED** [review]

**Review by Rajharsh2 — APPROVED** [review]
lgtm

**Pull request merged**